ThemeDecorator/ThemeDecorator.module.less

```diff
@@ -72,17 +72,20 @@
 		font-variant-ligatures: @agate-base-font-ligatures;
 		color: @agate-text-color;
 
-		&.bg {
-			background-color: @agate-bg-color;
-			background-image: @agate-bg-image;
-
-			&::before {
-				content: "";
-				display: block;
-				background-image: @agate-bg-image2;
-				position: absolute;
-				.position(0);
-			}
+		&.bg ::before {
```

Comment:
We usually don't put a space for pseudo classes and pseudo elements. Would you change this?